src/test/java/nearsoft/academy/bigdata/recommendation/MovieRecommender.java

```diff
@@ -0,0 +1,111 @@
+package nearsoft.academy.bigdata.recommendation;
+
+import org.apache.commons.collections.BidiMap;
+import org.apache.commons.collections.bidimap.TreeBidiMap;
+import org.apache.mahout.cf.taste.common.TasteException;
+import org.apache.mahout.cf.taste.impl.model.file.FileDataModel;
+import org.apache.mahout.cf.taste.impl.neighborhood.ThresholdUserNeighborhood;
+import org.apache.mahout.cf.taste.impl.recommender.GenericUserBasedRecommender;
+import org.apache.mahout.cf.taste.impl.similarity.PearsonCorrelationSimilarity;
+import org.apache.mahout.cf.taste.model.DataModel;
+import org.apache.mahout.cf.taste.neighborhood.UserNeighborhood;
+import org.apache.mahout.cf.taste.recommender.RecommendedItem;
+import org.apache.mahout.cf.taste.recommender.UserBasedRecommender;
+import org.apache.mahout.cf.taste.similarity.UserSimilarity;
+
+import java.io.*;
```

Comment:
Do you know the disadvantages of using wildcard imports? Why not use only what you need?

Reply:
Thank you, I corrected it, I was reading that it could lead to possible name conflicts and the readability of the code would decrease because the developers would not have a clear idea of the actual classes used.